src/node.ts

```diff
@@ -98,9 +106,21 @@ export function register(
     }
     return js
   }
-  installSourceMapSupport()
-  patchCommonJsLoader(compile)
-  addHook(compile, {
-    exts: extensions,
-  })
+
+  return {
+    enabled(enabled: boolean) {
+      const revert = addHook(compile, {
+        exts: extensions,
+      })
+
+      if (enabled === true) {
+        installSourceMapSupport()
+        patchCommonJsLoader(compile)
+      } else {
+        revert()
+      }
+    },
+  }
 }
```

Comment:
It actually breaks the [api](https://github.com/egoist/esbuild-register#programmatic-usage), I think `register` could just return the `revert` function:

```ts
const unregister = require('esbuild-register/dist/node').register()

unregister()
```

Reply:
Okay, let me make that change :D